I'm getting this error:

```
⇒  make test
GOBIN=/Users/mattbutcher/Code/Go/src/k8s.io/helm/bin go install  -tags '' -ldflags ' -X k8s.io/helm/pkg/version.Version=v2.0.0 -X k8s.io/helm/pkg/version.GitCommit=54f10f8ddda15cd2b24d5dee48fae215399b65ac -X k8s.io/helm/pkg/version.GitTreeState=dirty' k8s.io/helm/cmd/...

==> Running static validations <==
cmd/helm/fetch_test.go:1::warning: file is not gofmted with -s (gofmt)

==> Running linters <==
pkg/storage/storage.go:142:1:warning: exported method Storage.Last should have comment or be unexported (golint)
pkg/chartutil/chartfile.go:30:7:warning: const ApiVersionV1 should be APIVersionV1 (golint)
pkg/repo/repo.go:54:6:warning: type name will be used as repo.RepoFile by other packages, and that stutters; consider calling this File (golint)
make: *** [test-style] Error 1
☃ @ENG001590.local ~/Code/Go/src/k8
```

I think it just needs a `gofmt` run on `fetch_test.go`.

--- a/cmd/helm/fetch.go
+++ b/cmd/helm/fetch.go
@@ -124,7 +124,7 @@ func (f *fetchCmd) run() error {
 	}
 
 	if f.verify {
-		fmt.Fprintf(f.out, "Verification: %v", v)
+		fmt.Fprintf(f.out, "Verification: %v\n", v)
 	}
 
 	// After verification, untar the chart into the requested directory.

--- a/cmd/helm/fetch_test.go
+++ b/cmd/helm/fetch_test.go
@@ -18,8 +18,10 @@ package main
 
 import (
 	"bytes"
+	"fmt"
 	"os"
 	"path/filepath"
+	"regexp"
 	"testing"
 
 	"k8s.io/helm/pkg/repo/repotest"
@@ -39,13 +41,14 @@ func TestFetchCmd(t *testing.T) {
 
 	// all flags will get "--home=TMDIR -d outdir" appended.
 	tests := []struct {
-		name       string
-		chart      string
-		flags      []string
-		fail       bool
-		failExpect string
-		expectFile string
-		expectDir  bool
+		name         string
+		chart        string
+		flags        []string
+		fail         bool
+		failExpect   string
+		expectFile   string
+		expectDir    bool
+		expectVerify bool
 	}{
 		{
 			name:       "Basic chart fetch",
@@ -72,10 +75,11 @@ func TestFetchCmd(t *testing.T) {
 			fail:       true,
 		},
 		{
-			name:       "Fetch and verify",
-			chart:      "test/signtest",
-			flags:      []string{"--verify", "--keyring", "testdata/helm-test-key.pub"},
-			expectFile: "./signtest-0.1.0.tgz",
+			name:         "Fetch and verify",
+			chart:        "test/signtest",
+			flags:        []string{"--verify", "--keyring", "testdata/helm-test-key.pub"},
+			expectFile:   "./signtest-0.1.0.tgz",
+			expectVerify: true,
 		},
 		{
 			name:       "Fetch and fail verify",
@@ -87,16 +91,17 @@ func TestFetchCmd(t *testing.T) {
 		{
 			name:       "Fetch and untar",
 			chart:      "test/signtest",
-			flags:      []string{"--verify", "--keyring", "testdata/helm-test-key.pub", "--untar", "--untardir", "signtest"},
+			flags:      []string{"--untar", "--untardir", "signtest"},
 			expectFile: "./signtest",
 			expectDir:  true,
 		},
 		{
-			name:       "Fetch, verify, untar",
-			chart:      "test/signtest",
-			flags:      []string{"--verify", "--keyring", "testdata/helm-test-key.pub", "--untar", "--untardir", "signtest"},
-			expectFile: "./signtest",
-			expectDir:  true,
+			name:         "Fetch, verify, untar",
+			chart:        "test/signtest",
+			flags:        []string{"--verify", "--keyring", "testdata/helm-test-key.pub", "--untar", "--untardir", "signtest"},
+			expectFile:   "./signtest",
+			expectDir:    true,
+			expectVerify: true,
 		},
 	}
 
@@ -125,6 +130,15 @@ func TestFetchCmd(t *testing.T) {
 			}
 			t.Errorf("%q reported error: %s", tt.name, err)
 			continue
+		}
+		if tt.expectVerify {
+			pointerAddressPattern := "0[xX][A-Fa-f0-9]+"
+			sha256Pattern := "[A-Fa-f0-9]{64}"
+			verificationRegex := regexp.MustCompile(
+				fmt.Sprintf("Verification: &{%s sha256:%s signtest-0.1.0.tgz}\n", pointerAddressPattern, sha256Pattern))
+			if !verificationRegex.MatchString(buf.String()) {
+				t.Errorf("%q: expected match for regex %s, got %s", tt.name, verificationRegex, buf.String())
+			}
 		}
 
 		ef := filepath.Join(outdir, tt.expectFile)

